tests/kustomize_build.sh

```diff
@@ -21,20 +21,6 @@ DIR="$(dirname "$0")"
 # shellcheck source=tests/common.sh
 source "$DIR"/common.sh
 
-############################
-# Docker Tests
-############################
-[[ -z "${NODOCKER}" ]] || {
-  echo "Skipping docker tests"
-  exit 0
-}
-
-# TODO: Revisit after adressing https://github.com/GoogleContainerTools/kpt/issues/839
-testcase "docker_kustomize_build_imperative_git"
-docker run -u "$(id -u)" gcr.io/kpt-functions/kustomize-build:"${TAG}" -d path=https://github.com/kubernetes-sigs/kustomize/examples/multibases/ |
-  docker run -i -u "$(id -u)" -v "$(pwd)":/sink gcr.io/kpt-functions/write-yaml:"${TAG}" -o /dev/null -d sink_dir=/sink -d overwrite=true
-assert_contains_string pod_cluster-a-staging-myapp-pod.yaml "name: cluster-a-staging-myapp-pod"
-
 ############################
 # kpt fn Tests
 ############################
```

Comment:
Remove these headers as they're redundant. In other files too.

Reply:
Done